test/basic.test.tsx

```diff
@@ -1,82 +1,75 @@
-import * as React from 'react';
-import Downshift, { ControllerStateAndHelpers, StateChangeOptions } from '../';
+import * as React from 'react'
+import Downshift, { StateChangeOptions, DownshiftInterface } from '../'
+
+type Item = string
+const TypedDownShift: DownshiftInterface<Item> = Downshift
```

Comment:
I believe it will be great to have an example of simple usage of Downshift, like it was used before, without supplying type arguments. Just to ensure that the old way still works.

Reply:
Add example of simple usage of Downshift in `Custom.tsx`. Or you want to add one more file?

Reply:
Great! Thank you! Now we have examples of both typed and untyped versions.